pkg/kgo/consumer.go

```diff
@@ -758,6 +758,32 @@ func (cl *Client) AddConsumeTopics(topics ...string) {
 	cl.triggerUpdateMetadataNow("from AddConsumeTopics")
 }
 
+// GetConsumeTopics retrives a list of current topics to be consumed.
+//
+// Note that If you are directly consuming and specified ConsumePartitions,
+// this function will not be aware of the consumption topic target(s).
```

Comment:
wdym this isn't aware of partition consuming. I think looking at the code, `tps` is added to for partition consuming as well.

(also If => if)

Reply:
Upon review I believe you to be correct. I'll amend the commit.
(I'll also fix the if XD )

Reply:
This has been completed.